app/src/main/res/values/strings.xml

```diff
@@ -5,8 +5,8 @@
     <string name="action_search">Search</string>
     <string name="action_add">Add</string>
 
-    <string name="noGiftCards">You don\'t have any cards at the moment. Click the "+" (plus) button to get started.\n\nCatima lets you carry your cards on your phone, so they are always within reach.</string>
-    <string name="noMatchingGiftCards">No cards match the filter. Please try some different terms.</string>
+    <string name="noGiftCards">Click the "+" (plus) button to add a card first.\n\nCatima carries your cards on the device, so they are always within reach.</string>
+    <string name="noMatchingGiftCards">Could not find any such cards. Please try changing your search.</string>
```

Comment:
It's not only a search anymore though, it could also be that there are no cards in the selected groups.

But I do agree this is an improvement.